test/unit/SupSub.test.js

```diff
@@ -158,10 +158,10 @@ suite('SupSub', function() {
       assert.equal(mq.latex(), 'x_a^b');
 
       mq.keystroke('Left Backspace');
-      assert.equal(mq.latex(), 'xb_a');
+      assert.equal(mq.latex(), 'x_ab');
```

Comment:
This makes sense if you look at the latex directly, but it seems like an odd user experience that something would move from the exponent to the subscript when you press the delete key. It feels like in this case we should maybe move the cursor without changing structure.